packages/contracts-sdk/src/lib/contracts-sdk.ts

```diff
@@ -1667,7 +1677,7 @@ https://developer.litprotocol.com/v3/sdk/wallets/auth-methods/#auth-method-scope
             break;
           }
           tries++;
-          await new Promise((resolve, _reject) => {
+          await new Promise((resolve) => {
```

Comment:
Not a blocker for this PR, as I know this wasn't changed in this PR, but it occurs to me that on the new chain we might be able to change this to retry a lot sooner than 10 seconds 🤔

Reply:
Seems like this could be moved into a chain configuration constant